archstrike/rekall/PKGBUILD

```diff
@@ -0,0 +1,41 @@
+# Maintainer: ArchStrike <team@archstrike.org>
+# Contributor: strayArch <j.a.stronz@gmail.com>
+
+buildarch=1
+
+pkgname=rekall
+pkgver=1.6.0
+pkgrel=1
+pkgdesc="A Memory Forensic Framework"
+url="http://www.rekall-forensic.com"
+arch=('any')
+groups=('archstrike' 'archstrike-analysis' 'archstrike-forensics')
+license=('GPL2')
+depends=('python2')
+makedepends=('git' 'python2' 'python2-setuptools' 'python2-pip' 'python2-virtualenv')
+source=()
```

Comment:
I attempted to make the pkgbuild a git vcs. however, cryptography always failed to build unless I install into the env using pip. It isn't ideal, but they freeze the pip requirements and it's kind of a headache. Open to suggestions here.